seacrowd/sea_datasets/visim_400/visim_400.py

```diff
@@ -0,0 +1,181 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+ViSim-400 is a Vietnamese dataset of semantic relation \
+    pairs for evaluation of models that reflect the \
+    continuum between similarity and relatedness.
+"""
```

Comment:
I'm assuming you want to add module docs to this file, right? IMO it's unnecessary as we already describe it in the `DESCRIPTION`. But it's okay if you want to maintain it as part of your 'style'.

Reply:
I just follow the instructions on `/templates/template.py` here:

https://github.com/SEACrowd/seacrowd-datahub/blob/94bb4a18f345ef773a6922ce119005a74f0bdaa1/templates/template.py#L17-L29

Specifically on this line:

https://github.com/SEACrowd/seacrowd-datahub/blob/94bb4a18f345ef773a6922ce119005a74f0bdaa1/templates/template.py#L29

It seems, that since it doesn't impact the dataloader, I just need to resolve this conversation.

Reply:
For the consistency with the rest of the submitted and approved dataloaders, I solved in https://github.com/SEACrowd/seacrowd-datahub/pull/608/commits/0eed3234c48c3b91f8af647cf1256666e8133b3f.